ui/app/auth/options.ts

```diff
@@ -0,0 +1,38 @@
+import { Configuration } from '@/app/config/config';
+import { AuthOptions } from 'next-auth';
+import CredentialsProvider from 'next-auth/providers/credentials';
+import { Provider } from 'next-auth/providers/index';
+
+function getEnabledProviders(): Provider[] {
+  return [
+    CredentialsProvider({
+      name: 'Password',
+      credentials: {
+        password: { label: 'Password', type: 'password' },
+      },
+      async authorize(credentials, req) {
+        if (
+          credentials == null ||
+          credentials.password != Configuration.authentication.PEERDB_PASSWORD
+        ) {
+          return null;
+        }
+        return { id: '1', name: 'Admin' };
```

Comment:
what is this for?

Reply:
We need to return a `User` which requires an `id` in order to tell next-auth that we have a valid user. I just added a name just in case.